docs/coverage-frequency.md

```diff
@@ -0,0 +1,91 @@
+# Action frequency
+
+When adding coverage for a new action to `SimulController`, you'll need to define its relative frequency with respect to other actions: is the new action as frequent as, let's say, creating a post, or is it more similar to updating the custom status? This number is used by the controller to choose which action to perform, so one with a value of ten is two times more likely to happen than one with a value of five. But how does one come up with such a number?
+
+There's actually not a 100% right way to do it, since ideally we would like to model all types of Mattermost servers out there, and they all differ from each other. Even the same server may show different frequencies now to the ones seen one year ago. However, we can try our best and at least be internally consistent, so the frequencies of all new actions should be calculated following this guide.
+
+Go through the following section for a short bullet list of what's needed, or read the Detailed guide section if you want a more thorough explanation.
+
+## Summary
+
+In short, this whole process can be summarized as follows:
+
+1. Log in to [grafana.internal.mattermost.com](https://grafana.internal.mattermost.com).
+2. Choose the two-hour window with the most traffic for the past seven days.
+3. Run the following query in that window:
+```promql
+sum(rate(mattermost_api_time_count{handler=~"the-name-of-your-handler"}[2h]))
+/
+sum(rate(mattermost_api_time_count{handler=~"createPost"}[2h]))
+```
+4. Get the last value at the very end of the window.
+5. Multiply that value by 1.5 (the frequency of `createPost`). That's your frequency!
+
+## Detailed guide
+
+The frequencies assigned to actions in this controller are extrapolated from real frequencies seen in the Community server, so when adding a new one, we need to look at data from [community.mattermost.com](https://community.mattermost.com) to decide which frequency it'll have. This guide assumes that the feature was deployed to Community at least seven days prior. If you're adding coverage to the load-test before that, you'll have to simply *guess* how your action relates to the other ones, since there's no actual data out there to base your decision on. Once the feature is merged and a week has passed, you can follow this guide and hone that number in a second PR.
```

Comment:
```suggestion
The default frequencies assigned to actions in this controller are extrapolated from real frequencies seen in the Mattermost Community server, so when adding a new one, we need to look at data from [community.mattermost.com](https://community.mattermost.com) to decide which frequency it'll have. Alternatively, you can check your instance metrics to calculate frequencies. This guide assumes that the feature was deployed at least seven days before. If you're adding coverage to the load-test before that, you'll have to simply *guess* how your action relates to the other ones, since there's no actual data out there to base your decision on. Once the feature is merged and a week has passed, you can follow this guide and hone that number in a second PR.
```